tests/TestCase.php

```diff
@@ -11,6 +11,30 @@
 
 abstract class TestCase extends BaseTestCase
 {
+    public function expectException($exception)
+    {
+        if (method_exists('PHPUnit\Framework\TestCase', 'expectException')) {
+            return parent::expectException($exception);
```

Comment:
Since `expectException()` has a `void` return type, this will likely cause a notice with code inspection tools like PHPStan. I'd break this up and `return;` on the next line to steer clear of that.

Likewise for the methods below.